src/main/webapp/instructorHelp.jsp

```diff
@@ -86,49 +86,52 @@
         <div style="margin: 0 auto; padding: 0 50px;">
           <ul>
             <li>
-              <a href="#fbSetupSession">Setting up a feedback session</a>
+              <a data-toggle="collapse" data-target="#fbSetupSessionBody" href="#fbSetupSession">Setting up a feedback session</a>
             </li>
             <li>
-              <a href="#fbSetupQuestions">Setting up questions</a>
+              <a data-toggle="collapse" data-target="#fbCreateSessionBody" href="#fbCreateSession">Creating and scheduling a feedback session</a>
             </li>
             <li>
-              <a href="#fbPreview">Previewing a session</a>
+              <a data-toggle="collapse" data-target="#fbSetupQuestionsBody" href="#fbSetupQuestions">Setting up questions</a>
             </li>
             <li>
-              <a href="#fbViewResults">Viewing results</a>
+              <a data-toggle="collapse" data-target="#fbPreviewBody" href="#fbPreview">Previewing a session</a>
             </li>
             <li>
-              <a href="#fbQuestionTypes">Question Types</a>
+              <a data-toggle="collapse" data-target="#fbViewResultsBody" href="#fbViewResults">Viewing results</a>
+            </li>
+            <li>
+              <a data-toggle="collapse" data-target="#fbQuestionTypesBody" href="#fbQuestionTypes">Question Types</a>
               <ul>
                 <li>
-                  <a href="#fbEssay">Essay question</a>
+                  <a data-toggle="collapse" data-target="#fbEssayBody" href="#fbEssay">Essay question</a>
```

Comment:
Clicking on these questions doesn't link to the question details in the page itself. It opens up the sub-section in the "What types of questions can I add to a session?" section but doesn't link to it unless the section is open.

Reply:
Yup, this was just a weird (not very functional) workaround -- the hope is to get rid of the table of contents entirely